### DISCUSSION TITLE
Issue with loading plugin assets when Freeform in a headless environment

### DISCUSSION THREAD

**Discussion by fredrik-sogaard** [solspace/craft-freeform#521] [Questions & Answers] [answered]
~~I'm running Craft in a headless environment and thus have my `siteUrl` pointed to a NextJS frontend.~~

~~When I create forms and render them in a normal Twig template in the Craft installation, Freeform tries to load `plugin.js` and `plugin.css` with `siteUrl` prepended—which in this case is the NextJS frontend, and it can't find those files.~~

~~Is there a way to load the files manually or override the location of these files?~~

~~Using `craft.freeform.loadFreeformPlugin()` has the same problem.~~

Edit: I figured this out. You can simple link to the script file like so `<script src="/freeform/plugin.js"></script>`

**Comment by kjmartens** [comment, marked as answer]
Hi @fredrik-sogaard,

Yes, you can also add direct links to the scripts like: 🙂 

``` html
<script type="text/javascrpt" src="{siteUrl}/freeform/plugin.js"></script>
<link rel="stylesheet" href="{siteUrl}/freeform/plugin.css" />
```
